tasks/run_cd4pe_job.rb

```diff
@@ -378,9 +379,19 @@ def set_job_env_vars(task_params)
     end
 end
 
-def make_working_dir(working_dir)
-  @logger.log("Creating working directory #{working_dir}.")
-  Dir.mkdir(working_dir) unless File.exists?(working_dir)
+def make_dir(dir)
+  @logger.log("Creating directory #{dir}")
```

Comment:
nit: should we only log this if we're actually creating the dir?

Reply:
Yes and no. We say "creating directory!" and then immediately say "directory already exists!" if it already exists, and "Successfully created directory" if we create it.

I think this is more communicative than simply printing "directory already exists!"